java/runtime/src/main/java/org/ray/runtime/context/LocalModeWorkerContext.java

```diff
@@ -5,10 +5,11 @@
 import org.ray.api.id.JobId;
 import org.ray.api.id.TaskId;
 import org.ray.api.id.UniqueId;
+import org.ray.runtime.config.RunMode;
 import org.ray.runtime.generated.Common.TaskSpec;
 import org.ray.runtime.generated.Common.TaskType;
 import org.ray.runtime.task.LocalModeTaskSubmitter;
-import sun.reflect.generics.reflectiveObjects.NotImplementedException;
+import org.apache.commons.lang3.NotImplementedException;
```

Comment:
How about using java.lang.UnsupportedOperationException instead? The message does say "Not supported" so it's arguably the better message.

Reply:
Sounds good. I created another PR to replace all the `NotImplementedException`s. #5694

Reply:
Oops. Only one `NotImplementedException`s found after merging master (previously three). Either updating and merging this PR or merging #5694 is OK.